src/api/httpClient.tsx

```diff
@@ -63,7 +63,7 @@ async function resErrInterceptor(error: AxiosError<ErrorResponse>) {
         if ('error' in res) {
           store.dispatch(logout());
           httpClient.defaults.headers.common['Authorization'] = '';
-          toast.error('Your session has expired, please sign in again');
+          toast.error(`${i18n.t(ToastMessages.ErrorSessionExpired)}`);
```

Comment:
The `t` function returns a string. Why do you wrap it in a template string?

Check this issue across all of your changes